updatecli/updatecli.d/Mogztter-kroki.yml

```diff
@@ -0,0 +1,59 @@
+---
+title: "Bump Asciidoctor-Kroki version"
+sources:
+  getAsciidoctorKrokiVersion:
+    kind: githubRelease
+    name: "Get the latest Asciidoctor-Kroki version"
+    spec:
+      owner: "Mogztter"
+      repository: "asciidoctor-kroki"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionFilter:
+        kind: latest
+    transformers:
+      - trimPrefix: "v"
+conditions:
+  testDockerfileArgVersion:
+    name: "Does the Dockerfile have an ARG instruction which key is asciidoctor_kroki_version?"
+    kind: dockerfile
+    spec:
+      file: Dockerfile
+      instruction:
+        keyword: "ARG"
+        matcher: "asciidoctor_kroki_version"
+targets:
+  updateHarnessEnvAsciidoctorKrokiVersion:
+    name: "Update the key ASCIIDOCTOR_KROKI_VERSION in the test harness env_vars.yml file"
+    kind: yaml
+    sourceID: getAsciidoctorKrokiVersion
+    spec:
+      file: "tests/env_vars.yml"
+      key: "ASCIIDOCTOR_KROKI_VERSION"
```

Comment:
You need to add the line `ASCIIDOCTOR_KROKI_VERSION: 0.5.0` in the file `tests/env_vars.yml` to allow:

- Exposing the variable `ASCIIDOCTOR_KROKI_VERSION` to the test harness
- Injecting automatically the variable `ASCIIDOCTOR_KROKI_VERSION` in the generated README.adoc to ensure that it's up to date.

Sorry for this behavior, we'll simplify it soon, but it's needed.

Reply:
yeah, I realized that I missed this :) after I pushed the changes. I will update it